### PR TITLE
Use ThriftSender in JaegerExporter

### PR DESCRIPTION
**Pull request by MichaHoffmann**
Addresses #1798

### CODE REVIEW

**Comment by songy23** [comment]
Please sign the CLA.

**Comment by MichaHoffmann** [comment]
Sorry, i have to wait for my company before i sign.

**Comment by CyrenGmbHOwner** [comment]
I signed it!

**Comment by MichaHoffmann** [comment]
I signed it!

**Comment by songy23** [comment]
Ping @googlebot

**Comment by songy23** [comment]
Hi @MichaHoffmann it seems the author in the commit is a different username (mhoffmann). Can you try rebasing and push again?

Also please make sure the email associated with your Github account is the one that signed CLA.

**Comment by MichaHoffmann** [comment]
Hey @songy23, thanks i didnt notice!

File: exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerTraceExporter.java [hunk 1]
```diff
@@ -72,15 +73,15 @@ public static void createAndRegister(final String thriftEndpoint, final String s
...
    * HttpSender. Only one Jaeger exporter can be registered at any point.
    *
-   * @param httpSender the pre-configured HttpSender to use with the exporter
+   * @param sender the pre-configured ThriftSender to use with the exporter
    * @param serviceName the local service name of the process.
    * @throws IllegalStateException if a Jaeger exporter is already registered.
    * @since 0.17
    */
-  public static void createWithSender(final HttpSender httpSender, final String serviceName) {
+  public static void createWithSender(final ThriftSender sender, final String serviceName) {
```
Comment (songy23) [inline exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerTraceExporter.java]: Will this be backwards-compatible? If not consider adding a separate `createWithSender` API that takes `ThriftSender`.

Comment (MichaHoffmann) [reply exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerTraceExporter.java]: HttpSender extends ThriftSender if i remember correctly; One should be able to just supply a HttpSender to the Method.

https://github.com/jaegertracing/jaeger-client-java/blob/20ee4e81749bbd0037389d04c32cfd296b3857d3/jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/HttpSender.java#L34

**Pull request merged**